### DISCUSSION TITLE
Vercel CLI: "Error! SAML Single Sign-On is not enabled for this Team"

### DISCUSSION THREAD

**Discussion by acomito** [vercel/vercel#6159] [Help] [answered]
I can't seem to find an SSO option to enable for this and google didn't show anyone else running into this issue. So, I can't login via the CLI.

I'm wondering if it's because my account is so old (created in the early days of zeit i think). Or am I missing where this setting is in teams?

**Comment by paulogdm** [comment]
Where are you observing this error? Are you a member of a team that has SAML enabled?

**Comment by TooTallNate** [comment, marked as answer]
Are you attempting to log in with the CLI? Can you share the exact command you are trying to run? Also, which version of CLI are you running? Thanks!

**Reply by omnibrain** [reply]
I'm experiencing the same problem when I try to log in with the vercel cli:

```bash
Vercel CLI 22.0.1 dev (beta) — https://vercel.com/feedback
> > No existing credentials found. Please log in:
? Enter your email or team slug: <team name...>
Error! SAML Single Sign-On is not enabled for this Team
```

In the browser window that opens I just see this:

![image](https://user-images.githubusercontent.com/2551508/116524002-278e3c00-a8d7-11eb-8945-8f85a9a9fc85.png)

**Reply by icrotz** [reply]
I get the same error

**Reply by TooTallNate** [reply]
SAML Single Sign-On is an enterprise feature, so unless the team you are typing in is an enterprise Team, then you will encounter this error. Instead, you should type in your _email address_.

If you are actually trying to enable the SAML Single Sign-On feature for a Team, then you can get in touch with the [Sales](https://vercel.com/contact/sales) team.
